composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "humanmade/analytics",
+    "description": "Analytics module for Platform",
+    "type": "package",
+    "require": {
+        "humanmade/hm-gtm": "1.1.0"
```

Comment:
Presuming this needs adding to core too?

Reply:
PR at humanmade/platform-core#13